### PR TITLE
Add file to summarize results for the presentation

### PR DESCRIPTION
**Pull request by etrieschman**
Hi! I just created a file to make tables and figures for our presentation. I wrote `@kamran` in different places in the code to call out where you can change dictionaries for the final results. So far i made the tables in my section, as well as the topline table in your section. I haven't made the rugplot b/c i wasn't sure how far along you were with that.

to generate the final results, I think we can hopefully just run this code after we've run the 8760 networks on the cluster (thanks a lot for taking care of that, btw).

hope you're having a great weekend!

### CODE REVIEW

File: src/scripts/make_presentation_outputs.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                            network='rts_gmlc_full', nhours=24,
+                            X=X, l='4.0', m='particle')
+
+# get iterations
+iter_options = {'network':snakemake.config['networks'],
+                'input_sample':snakemake.config['GP_Model']['profile_type'],
+                'nhours':snakemake.config['solving']['options']['nhours'], 
+                'methods':snakemake.config['GP_Model']['sampling_methods']}
+
+# @Kamran -- this is where we can "cherrypick the characteristic length" for each method
```
Comment (etrieschman) [inline src/scripts/make_presentation_outputs.py:80]: @ktehranchi

File: src/scripts/make_presentation_outputs.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                plt.savefig(RESULTS + f'temp_for_presentation/dist_{n}_{X}_{hr}hr.png', dpi=300, bbox_inches='tight')
+                # CALCULATE DIFFERENTIAL STATISTICS
+                fpred_bootstrap_results = dict_list()
+                for fpred in fpreds_scale.T:
+                    get_stats(fpred_bootstrap_results, fpred)
+                diff_results = get_diffs(fscale_results, fpred_bootstrap_results)
+                for k in diff_results.keys():    
+                    summ[(n, hr, X, m, k)] = [diff_results[k]['mean'], diff_results[k]['std']]
+
+                # @kamran -- you could include plots for the index values within this for loop if its easy``
```
Comment (etrieschman) [inline src/scripts/make_presentation_outputs.py:121]: @ktehranchi

File: src/scripts/make_presentation_outputs.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+# format table
+summ_df = pd.DataFrame(summ).T.reset_index()
+summ_df.columns = ['network', 'n_hours', 'profile_type', 'method', 'stat', 'mean', 'std']
+summ_df['value'] = summ_df['mean'].round(3).astype(str) + '±' + summ_df['std'].round(3).astype(str)
+summ_df = summ_df.pivot(index=['network', 'n_hours', 'stat'], columns=['method', 'profile_type'], values=['value'])
+summ_df.reset_index().to_csv(RESULTS + f'temp_for_presentation/table_diff.csv', index=False)
+
+# %%
+# SAVE QUANTITATIVE COMPARISION OF DISTRIBUTIONS (best network)
+# @KAMRAN -- HERE'S THE CODE FOR THE TOPLINE RESULTS TABLE
```
Comment (etrieschman) [inline src/scripts/make_presentation_outputs.py:135]: @ktehranchi

**Comment by ktehranchi** [comment]
Great! I just got the cluster compute working.... ill send results later today and will add the rug plots/summary tables as well.

**Review by ktehranchi — APPROVED** [review]

**Pull request merged**